tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs

```diff
@@ -17,6 +17,7 @@
 using Windows.Media.Protection;
 using Windows.Storage.Pickers;
 using WinUIEx;
+using YamlDotNet.Core.Tokens;

```

Comment:
Did I miss what we are using from this new namespace?

Reply:
I deleted this in local. Was waiting for comments before pushing out